lib/phoenix_component/declarative.ex

```diff
@@ -305,6 +313,10 @@ defmodule Phoenix.Component.Declarative do
       compile_error!(line, file, "global attributes do not support the :examples option")
     end
 
+    if type != :global and Keyword.has_key?(opts, :include) do
```

Comment:
should this be a warning? it won't break anything per-say if someone does make an oopsie.

Reply:
it's not valid so we should raise